### PR TITLE
017 axis ticks

### PR DESCRIPTION
**Pull request by aknipey**
Added axis ticks  and minor gridlines for x and y for main graph, and a line at x=0 on the distribution
Screenshot from 2023-09-19 23-23-09
Screenshot from 2023-09-19 23-23-21

### CODE REVIEW

File: frontend/src/components/OutcomeGraph.tsx [hunk 1]
```diff
@@ -17,6 +17,18 @@ import {
 
 const DISPLAY_MARKERS: number[] = [0.1, 0.5, 0.75, 0.9];
 
+const MINOR_GRID_STYLE = {
```
Comment (patrickemoore) [inline frontend/src/components/OutcomeGraph.tsx:20]: type

File: frontend/src/components/OutcomeGraph.tsx [hunk 2]
```diff
@@ -17,6 +17,18 @@ import {
...
+  tickcolor: "black",
+  nticks: 4,
+  showgrid: true,
+  showline: true,
+  mirror: "ticks",
+  gridwidth: 2,
+  minorgridwidth: 3,
+};
+
 const INDEPENDENT_GRAPH_RET = {
```
Comment (patrickemoore) [inline frontend/src/components/OutcomeGraph.tsx:32]: type

File: frontend/src/components/OutcomeGraph.tsx [hunk 3]
```diff
@@ -210,7 +223,8 @@ export const OutcomeGraph = memo(function OutcomeGraph(props: {
 
   const graphLayout = useMemo(() => {
```
Comment (patrickemoore) [inline frontend/src/components/OutcomeGraph.tsx:224]: type

File: frontend/src/components/ParameterInput.tsx [hunk 4]
```diff
@@ -443,6 +443,7 @@ export const ParameterInput = memo(function ParameterInputComp({onClick, rerende
           <OutcomeGraph simulation={simulation as Simulation} />    
       
         </Grid2>
+        <br />
```
Comment (patrickemoore) [inline frontend/src/components/ParameterInput.tsx:446]: i am generally against <br /> prefer margin/stack spacing

**Review by patrickemoore — APPROVED** [review]
lgtm, see inline for f'up

File: frontend/src/components/OutcomeGraph.tsx (hunk 2 above)
Comment (aknipey) [reply frontend/src/components/OutcomeGraph.tsx:32]: typed in the sidebar pr

**Pull request merged**